packages/teams-js/src/public/authentication.ts

```diff
@@ -202,6 +195,12 @@ export namespace authentication {
     });
   }
 
+  function throwIfStringNotValidHttpsUrl(link: string): void {
+    if (!isValidHttpsURL(new URL(link))) {
+      throw new Error(`${ErrorCode.INVALID_ARGUMENTS}: url must be https`);
```

Comment:
Would it be appropriate to throw an SdkError and include the `ErrorCode.INVALID_ARGUMENTS` as the ErrorCode parameter?

Reply:
No, honestly I don't think we should keep throwing SdkErrors. When we throw Error objects they come along with extra info like callstack that we don't get when throwing SdkErrors.

Reply:
Good to know, thanks!